### PR TITLE
Get help index page edits

### PR DESCRIPTION
**Pull request by rubymorillo**

### CODE REVIEW

**Comment by janl** [comment]
lgtm

File: get-help/index.html [hunk 1]
```diff
@@ -7,21 +7,18 @@
...
-        <h2>For any question you may have</h2>
-        <ul>
-            <li>Check our <a href="http://faq.hood.ie" target="_blank">FAQ</a> or</li>
-            <li><a href="https://github.com/hoodiehq/faq/issues/new" target="_blank">Send us a new question</a>. Having those questions in one place helps us make this content accessible for everyone in the community. If you don’t have a GitHub account, please <a href="/contact">contact us</a>.</li>
-            <li>Report a bug <a href="/bug">here</a>.</li>
-            <li>Maybe a <strong>Cheat Sheet</strong> can help you while you're getting familiar with Hoodie? You can get it here: <a href="/dist1/presentations/hoodie-cheat-sheet.pdf">Web Version</a> | <a href="/dist1/presentations/hoodie-cheat-sheet-print.pdf">Print Version</a>.
-        </ul>
+        <h2>Have questions about using Hoodie?</h2>
+        <p>
+        	We’re in the process of making our documentation even better, so if you have a question related to using Hoodie, join our public <a href="/chat">Slack</a> community and ask for help in the <strong>#help<strong> channel!
```
Comment (gr2m) [inline get-help/index.html]: the `<strong>` tag doesn’t close correctly. `<strong>#help<strong>` should be `<strong>#help</strong>`

**Comment by gr2m** [comment]
Looking good, can you just fix the one `strong` tag issue? Then it’s good to merge from my side

**Comment by rubymorillo** [comment]
Updated the strong tag, let me know if there are other issues! @gr2m @janl

**Comment by gr2m** [comment]
LGTM

**Comment by gr2m** [comment]
merged via a3d17d7 :ok_hand: :clap: :bouquet:  http://hood.ie/get-help/

**Pull request closed without merging**